t/54-oracle-sql-diff.t

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/perl
+
+use FindBin qw/$Bin/;
+use Test::More;
+use Test::SQL::Translator;
+use Test::Exception;
+use Data::Dumper;
+use SQL::Translator;
+use SQL::Translator::Diff;
+
+BEGIN {
+    maybe_plan(6, 'SQL::Translator::Producer::Oracle');
+}
+
+my $schema1 = $Bin.'/data/oracle/schema-1.5.sql';
+my $schema2 = $Bin.'/data/oracle/schema-1.6.sql';
+
+open my $io1, '<', $schema1 or die $!;
+open my $io2, '<', $schema2 or die $!;
+
+my ($yaml1, $yaml2);
+{
+    local $/ = undef;
+    $sql1 = <$io1>;
+    $sql2 = <$io2>;
+};
+
+close $io1;
+close $io2;
+
+my $s = SQL::Translator->new(from => 'Oracle');
+$s->parser->($s,$sql1);
+
+my $t = SQL::Translator->new(from => 'Oracle', debug => 1);
+$t->parser->($t,$sql2);
+
+my $d = SQL::Translator::Diff->new
+  ({
+    output_db => 'Oracle',
+    target_db => 'Oracle',
+    source_schema => $s->schema,
+    target_schema => $t->schema,
+    sqlt_args => {quote_identifiers => 1}
+   });
+
+
+my $diff = $d->compute_differences->produce_diff_sql || die $d->error;
+
+ok($diff, 'Diff generated.');
+
+warn "The diff is: " . Dumper($diff);
```

Comment:
I don't think `warn` in tests is good practice?

Reply:
Again, My bad, that was supposed to get removed. I will take care of it.